lib/Jsrt/ChakraCore.h

```diff
@@ -163,7 +163,7 @@ JsGetModuleHostInfo(
     _In_ JsModuleHostInfoKind moduleHostInfo,
     _Outptr_result_maybenull_ void** hostInfo);
 
-#ifndef NTBUILD
+#ifdef CHAKRACOREBUILD_
```

Comment:
I'm not sure when and where we had discussed this but I had brought this `ifdef` after a discussion. I can happily remove it.